server/database/seed.js

```diff
@@ -1,15 +1,69 @@
 const mongoose = require('mongoose');
 const db = require('./database.js');
 const models = require('./databaseModels.js');
+const productSizes = models.productSizes;
 const generateSeedData = require('./generateSeedData.js');
+const dbname = process.env.NODE_ENV === 'test' ? 'ikea-test' : 'ikea';
+const host = process.env.DB_HOST_ENV === 'docker' ? 'mongo:27017' : 'localhost';
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
```

Comment:
Nice!  Looks really clean!  

The only completely minor suggestions I can think of would be if you considered using a variable for the filepath? 
 That way if you ever change it, it only has to be updated in one place, to keep it D.R.Y.. 

And I wondered if you ended up needing the csv-write-stream npm package? Not sure, but it looks like it is not referenced so maybe it could be uninstalled.